JSDemos/Demos/Scheduler/AllDayPanelMode/description.md

```diff
@@ -0,0 +1,10 @@
+The **All day** Scheduler panel displays appointments that last all day or have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true**. To change the **All day** panel display mode, assign one of the following values to the [allDayPanelMode](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#allDayPanelMode) property:
+
+- *'all'*    
+Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that have the length equal to or more than 24 hours. The Scheduler does not show these appointments on the view.
```

Comment:
Stronger verb allows to shorten the sentence: "have the length equal to or more than 24 hours" -> "last 24 hours or longer".

More often than not, Ray replaces "show" with "display" when we talk about our controls and their elements. Also "in the view" reads to me a bit better than "on the view". But maybe both are okay. 

Some of these comments apply to other list items.

```suggestion
Displays all appointments that have the [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) property set to **true** and the ones that last 24 hours or longer. The Scheduler does not display these appointments in the view.
```